tests/installation/system_role.pm

```diff
@@ -35,6 +35,15 @@ sub change_system_role {
         }
         else {
             assert_and_click "system-role-$system_role";
+            # On aarch64 there are issue for vnc refresh, open and close help window to force a redraw.
+            if (is_aarch64) {
+                if (!check_screen("system-role-$system_role-selected")) {
+                    send_key 'alt-h';
```

Comment:
IMO this should be a softfail with a poo# or boo# for further investigation. It's a product bug after all.

Reply:
Updated